functions.php

```diff
@@ -54,3 +56,71 @@ function displayDB(array $results) : string {
     return $dbResult;
 }
 
+
+/**
+ * Cleanse user input using filter_var() and return an array
+ *
+ * @param array $userInput user input in form is passed through
+ *
+ * @return array filtered user input collected in array and returned
+ */
+function cleanseData(array $userInput) : array {
+    $outputArr = [];
+    foreach($userInput as $strings){
+        if(!is_string($strings)){
+            return ['Incorrect input'];
+        }
+    }
+
+    foreach($userInput as $input){
+        $outputArr[] .= filter_var($input, FILTER_SANITIZE_SPECIAL_CHARS);
```

Comment:
this line can just be done in the foreach above as they are both looping over the same data